integration-libs/cdc/schematics/add-cdc/__snapshots__/index_spec.ts.snap

```diff
@@ -128,6 +130,22 @@ export class AdministrationWrapperModule { }
 "
 `;
 
+exports[`Spartacus CDC schematics: ng-add CDC feature general setup should install the appropriate dependencies 4`] = `
+"import { NgModule } from '@angular/core';
+import { CDCB2BRegisterModule } from "@spartacus/cdc/organization/user-registration";
+import { OrganizationUserRegistrationModule } from "@spartacus/organization/user-registration";
+
+@NgModule({
+  declarations: [],
+  imports: [
+    OrganizationUserRegistrationModule,
+    CDCB2BRegisterModule
```

Comment:
For consistency and better developer experience of customers, I suggest that the extending module should use the name of the original module + just add a custom `Cdc` prefix, e.g.:
```ts
imports: [
  OrganizationUserRegistrationModule,
  CdcOrganizationUserRegistrationModule
]
```
(instead of `CDCB2BRegisterModule`)

Side note: for readability let's keep the abbreviations CamelCased:
`CDCB2BRegisterModule` -> `CdcB2bRegisterModule`. That said, I still recommend rather aligning with the original name and adding a `Cdc` prefix